devices/power.py

```diff
@@ -58,6 +61,10 @@ def get_power_device(ip_address, username=None, password=None, outlet=None):
         return APCPower(ip_address, outlet=outlet)
     if '<b>IP9258 Log In</b>' in data:
         return Ip9258(ip_address, outlet, username=username, password=password)
+    if 'Access Error: Unauthorized' in data:
+        print("ip_address = %s" % ip_address)
```

Comment:
Is this data really unique? Can't we look for IP9820 text in the resulting output?